src/notation/internal/notationinteraction.cpp

```diff
@@ -82,7 +82,17 @@ using namespace mu::engraving;
 
 static int findSystemIndex(Ms::EngravingItem* element)
 {
-    Ms::System* target = element->parent()->isSystem() ? toSystem(element->parent()) : element->findMeasureBase()->system();
+    Ms::System* target = nullptr;
+    if (element->parent()->isSystem()) {
+        target = toSystem(element->parent());
+    } else if (const Ms::MeasureBase* mb = element->findMeasureBase()) {
+        target = mb->system();
+    }
+
+    if (!target) {
+        return -1;
```

Comment:
This code must be updated when https://github.com/musescore/MuseScore/pull/11281 is merged into the master

Reply:
To `return mu::invalid_index;` I guess